src/auth-service/models/Survey.js

```
@@ -0,0 +1,432 @@
+        updatedAt: 1,
+      };
+
+      const exclusionProjection = constants.SURVEYS_EXCLUSION_PROJECTION
+        ? constants.SURVEYS_EXCLUSION_PROJECTION(
+            filter.category ? filter.category : "none"
+          )
+        : {};
+
+      if (!isEmpty(filter.category)) {
+        delete filter.category;
+      }
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)